app/policies/case/base_policy.rb

```diff
@@ -302,6 +301,10 @@ def can_record_data_request?
     false
   end
 
+  def can_manage_offender_sar?
+    check_user_can_manage_offender_sar
+  end
+
```

Comment:
@Polsonby , sorry for asking this question again (you may have explained it to me like it will break some tests if not this level?),  I was wondering whether it is possible to not touch the base_policy level, but instead to overwrite this one and have the changes for function ```can_add_case``` in the offender_policy.rb level? I feel it would be more cleaner

Reply:
I see your point and I'd completely agree with you :)  - the thing is, though, at this point there isn't a case context - this is where the app has a completely generic user and has to figure out base abilities

![image](https://user-images.githubusercontent.com/1161161/83440326-a6b38f80-a43c-11ea-8e78-a4c62ef5d54e.png)